diagnostic.md

```diff
@@ -6,70 +6,72 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+Rails.application.routes.draw do
+  resources :books, only: [:new, :edit]
+  resources :books, except: [:new, :edit]
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+it runs a migration on the current database using the budle of gemfiles you have in that file
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+rails g scaffold creates all things accociated with things and rails g controller just creates a controller file
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+to define the attributes of the information in your table.
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```md
-// your response here
+We expect it to return the Patient hash
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
 ```md
-// your response here
+In the model
 ```
 
 Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+because the code they do hold is responsible for all the database interaction
```

Comment:
`The methods making use of the Patient model are defined within the controller.`